tempesta_fw/tfw_tfwcfg.h

```diff
@@ -0,0 +1,59 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2016 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+#include "str.h"
+#include "http_match.h"
+
+typedef tfw_http_match_op_t	tfw_match_t;
+
+/*
+ * List of Tempesta configuration directives. 
+ *
+ * "stmt" (statement) word is used to make
+ * ithe name of the variable short and concise.
+ */
+typedef enum {
+	/* Cache directives. */
+	TFW_D_CACHE_BYPASS,
+	TFW_D_CACHE_FULFILL,
+	TFW_D_CACHE_DEFAULT,
+
+	_TFW_D_COUNT
+} tfw_stmt_t;
```

Comment:
Cache bypass or fulfill is just a binary operation. It's unclear for me why do we need to handle 4-value enum instead of 2-value enum or just boolean type...